class UserAlreadyExistsError(Exception):
    pass


class WrongPassword(Exception):
    pass
